Alisha/data/lab3.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        sns.countplot(x=col, data=df, palette='pastel')
</pre><p>Add hue = 'colorvalue' to avoid deprecation warning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>